scripts/benchmarks/refs/a100_80gb.csv

```diff
@@ -1,49 +1,61 @@
+,,,,,none,10,,,,,2e-5,,,NousResearch/Llama-2-70b-hf,1,80837.0,,,8,,,,,,,,,,,,
+,,,,,none,11,,,,,2e-5,,,NousResearch/Llama-2-70b-hf,2,80834.5,,,4,,,,,,,,,,,,
+,16449040384.0,0.0,0.04,,none,12,-11098513408.0,12965134336.0,15047630848.0,0.0,2e-4,16,0.0,mistralai/Mistral-7B-v0.1,1,29731.0,26108963328.0,lora,4,16,q_proj k_proj v_proj o_proj,15119590912.0,0.9096682230631511,130080768.0,0.0,71960064.0,10989372416.0,136.624,0.878,0.22,3597.611
+,18520264704.0,512.0,0.04,,none,13,4096.0,0.0,0.0,0.0,2e-4,16,0.0,mistralai/Mistral-7B-v0.1,2,18697.0,15123161088.0,lora,2,16,q_proj k_proj v_proj o_proj,7850391552.0,0.8918854713439941,-13396860928.0,13398827008.0,7850391040.0,7272769536.0,82.0311,1.463,0.366,2995.936
+,17113726976.0,0.0,0.09,,none,14,-12773847040.0,13971894272.0,15047630848.0,0.0,2e-4,16,0.0,mistralai/Mistral-7B-v0.1,1,43195.0,37098695168.0,lora,8,16,q_proj k_proj v_proj o_proj,15119984128.0,0.962119706471761,582803456.0,966656.0,72353280.0,21978711040.0,270.6301,0.887,0.111,3632.412
+,18521251840.0,512.0,0.09,,none,15,65536.0,0.0,0.0,0.0,2e-4,16,0.0,mistralai/Mistral-7B-v0.1,2,26235.0,21433753600.0,lora,4,16,q_proj k_proj v_proj o_proj,7850588160.0,0.9218235015869141,-13390213120.0,13392883712.0,7850587648.0,13583165440.0,143.8184,1.669,0.209,3417.643
+,,,,,none,16,,,,,2e-4,16,0.0,mistralai/Mixtral-8x7B-Instruct-v0.1,1,80955.0,,lora,4,16,q_proj k_proj v_proj o_proj,,,,,,,,,,
+,97451773952.0,512.0,0.04,,none,17,4096.0,0.0,0.0,0.0,2e-4,16,0.0,mistralai/Mixtral-8x7B-Instruct-v0.1,2,62617.0,57540387840.0,lora,2,16,q_proj k_proj v_proj o_proj,47311452160.0,0.9361546834309896,-92310913024.0,92315766784.0,47311451648.0,10228935680.0,179.3128,0.669,0.167,1370.566
+,,,,,none,18,,,,,2e-4,16,0.0,mistralai/Mixtral-8x7B-Instruct-v0.1,1,80955.0,,lora,8,16,q_proj k_proj v_proj o_proj,,,,,,,,,,
+,97451958272.0,512.0,0.09,,none,19,4096.0,0.0,0.0,0.0,2e-4,16,0.0,mistralai/Mixtral-8x7B-Instruct-v0.1,2,69848.0,64347637760.0,lora,4,16,q_proj k_proj v_proj o_proj,47311648768.0,0.9383139928181966,-92302839808.0,92306579456.0,47311648256.0,17035988992.0,280.8919,0.854,0.107,1749.855
+,,,,,none,20,,,,,2e-4,16,0.0,NousResearch/Llama-2-70b-hf,1,80917.0,,lora,4,16,q_proj k_proj v_proj o_proj,,,,,,,,,,
+,,,,,none,21,,,,,2e-4,16,0.0,NousResearch/Llama-2-70b-hf,2,80894.0,,lora,2,16,q_proj k_proj v_proj o_proj,,,,,,,,,,
+,,,,,none,22,,,,,2e-4,16,0.0,NousResearch/Llama-2-70b-hf,1,80917.0,,lora,8,16,q_proj k_proj v_proj o_proj,,,,,,,,,,
+,,,,,none,23,,,,,2e-4,16,0.0,NousResearch/Llama-2-70b-hf,2,80979.0,,lora,4,16,q_proj k_proj v_proj o_proj,,,,,,,,,,
+/data/aaron/experimental/test3/scripts/benchmarks/../../sample-configurations/baseline-peft-bnb-nf4-sample-configuration.yaml,4515069952.0,5241734144.0,0.04,True,baseline-peft-bnb,24,8192.0,0.0,0.0,0.0,2e-4,16,0.0,mistralai/Mistral-7B-v0.1,1,27023.0,22825932800.0,lora,4,16,q_proj k_proj v_proj o_proj,5368221184.0,0.9589527130126954,403791872.0,0.0,126487040.0,17457711616.0,178.8061,0.671,0.168,2748.9
```

Comment:
need to understand why the full paths are being propagated in here. Previously we did not have them.